### DISCUSSION TITLE
Support chatGPT for editors?

### DISCUSSION THREAD

**Discussion by Stupidism** [payloadcms/payload#2099] [Feature Requests & Ideas]
https://www.youtube.com/watch?v=59hsPSrYvmk

**Comment by Stupidism** [comment]
Any thoughts?

**Reply by teunmooij** [reply]
@AlessioGr wrote a plugin that adds this functionality. I think it's part of this one: https://www.npmjs.com/package/payload-plugin-lexical

**Reply by AlessioGr** [reply]
Indeed, the lexical plugin tries to automatically complete your sentences if you enable it, though currently it does burn through your OpenAI credits - improvements regarding that are coming soon. Further ChatGPT integration into the editor is also planned (e.g. editor commands like "summarize XY", or "improve grammar")!

There's also an open PR for the SEO plugin to automatically generate Meta titles & descriptions
